proposals/dictionary-expressions.md

```diff
@@ -165,47 +165,48 @@ From [Open Question 3](#open-question-3), we will support *dictionary elements*
 2. Use applicable instance indexer if available; otherwise report an error during construction (or conversion?).
 3. Use C#12 initialization always.
 
+## Dictionary types
+
+A type is considered a *dictionary type* if the following hold:
+* The *iteration type* is `KeyValuePair<TKey, TValue>` and the *iteration type* is determined from a `GetEnumerator` instance method or enumerable interface.
```

Comment:
It seems like the latter part of this rule is blocking creatable but not directly enumerable dictionary types, is that intentional? What's the motivation for doing that? Since we just unblocked the analogous scenario for regular collection-exprs.

Reply:
Good catch, removed.